ihmc-perception/src/main/java/us/ihmc/perception/headless/HumanoidPerceptionModule.java

```diff
@@ -168,10 +176,8 @@ public void updateTerrain(ROS2Helper ros2Helper,
                                                             PerceptionAPI.HEIGHT_MAP_CROPPED,
                                                             croppedHeightMapImageMessage,
                                                             acquisitionTime);
-                                      //               publishHeightMapImage(ros2Helper, localHeightMapImage, compressedLocalHeightMapPointer, PerceptionAPI.HEIGHT_MAP_LOCAL,
-                                      //                                     localHeightMapImageMessage, acquisitionTime);
-                                      //               publishHeightMapImage(ros2Helper, globalHeightMapImage, compressedInternalHeightMapPointer, PerceptionAPI.HEIGHT_MAP_GLOBAL,
-                                      //                                     globalHeightMapImageMessage, acquisitionTime);
+                                      //publish a global height map
```

Comment:
Being picky but its good practice to format comments right above the line they influence, and as far as IHMC standard practices go comments have a space after the //. So it would look like `// Publish a global height map`